app/models/manageiq/providers/cloud_manager/virtual_template.rb

```diff
@@ -0,0 +1,14 @@
+class ManageIQ::Providers::CloudManager::VirtualTemplate < ::MiqTemplate
+  validate :single_template, :on => :create
+  default_value_for :cloud, true
+
+  TYPES = {
+    :amazon => 'ManageIQ::Providers::Amazon::CloudManager::VirtualTemplate'
+  }.freeze
+
+  def single_template
+    multiple = !type.constantize.where(:type => type).empty?
```

Comment:
@Fryguy Yeah, this is a temporary placeholder.

Right now with the way the API works, the only supported vendor is technically amazon. Ideally this will search on the `vendor` and not the `type`. This was more or less to be able to write specs for multiple template creation - this validation fails if the type is just the generic `ManageIQ::Providers::CloudManager:VirtualTemplate` and you search on its class - because then it returns all Virtual Templates. Which is why searching on the `type` solved this problem (that make sense?) 

Solution to the problem would just be making this change now and disabling multiple creation specs until later.